app/helpers/submissions_helper.rb

```diff
@@ -9,4 +9,8 @@ def eligible_for_evaluation?(submission)
   def selected_to_advance?(submission)
     submission.judging_status.in?(%w[winner])
   end
+
+  def available_evaluators(submission)
+    submission.phase.evaluators - submission.evaluators || []
```

Comment:
you don't need the || [] fallback right? `[] - [] == []`
```suggestion
    submission.phase.evaluators - submission.evaluators
```